tests/uefistored/conftest.py

```diff
@@ -1,31 +1,41 @@
+import logging
 import pytest
 
 @pytest.fixture(scope='module')
-def imported_sb_vm(imported_vm):
-    vm = imported_vm
+def pool_without_uefi_certs(host):
+    pool = host.pool
 
-    if not vm.is_uefi:
-        pytest.skip('imported_sb_vm can only be used on UEFI VMs')
+    # Save the certs.
+    # This may fail if the pool certs are not in a consistent state, and prevent the tests from running.
```

Comment:
I wonder if we should place this in the README too. A gentle reminder about using secureboot-certs to initialize a pool for tests, perhaps?

Reply:
You don't need to initialize the pool with secureboot-certs. The recorded state will just be that no certs are installed, so all certs will be deleted during teardown in this situation.

I've just added a Requirements comment on top of the file that says that the state must be consistent. As there is no way to enter an inconsistent state using secureboot-certs (in theory), I don't think it needs to be in the main README.